invenio_records_rest/utils.py

```diff
@@ -187,10 +192,18 @@ def data(self):
 
 
 class PIDConverter(BaseConverter):
-    """Resolve PID value."""
+    """Converter for PID values in the route mapping.
+
+    This class is a custom routing converter defining the 'PID' type.
+    See http://werkzeug.pocoo.org/docs/0.12/routing/#custom-converters.
+
+    Use ``pid`` as a type in the route pattern, e.g.: the use of
+    route decorator: ``@blueprint.route('/record/<pid:pid_value>')``,
```

Comment:
The converter requires at least one parameter (the pid_type). Also using the extra `getter` and `record_class` aguments is common. [Example from Zenodo](https://github.com/zenodo/zenodo/blob/master/zenodo/modules/deposit/views.py#L123-L127):

```python
@blueprint.route('/records/<pid(recid,record_class="zenodo.modules.records.api:ZenodoRecord"):pid_value>')
```